### PR TITLE
Add archive notice

### PR DESCRIPTION
**Pull request by fcollonval**
Follow-up of #25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # theme-cookiecutter
 
-[![Build Status](https://github.com/jupyterlab/theme-cookiecutter/workflows/CI/badge.svg)](https://github.com/jupyterlab/theme-cookiecutter/actions)
+## Archived
+
+**This project is archived. Development is now happening in https://github.com/jupyterlab/extension-cookiecutter-ts**.
```
Comment (jtpio) [inline README.md]:
```suggestion
**This project is archived. It is now possible to create a Theme Extension for JupyterLab using the following cookiecutter: https://github.com/jupyterlab/extension-cookiecutter-ts**.
```

**Review by jtpio — APPROVED** [review]
Thanks!

Left a minor suggestion to let extension authors know why they should be redirected to the main TS cookiecutter.

**Comment by fcollonval** [comment]
Thanks for the suggestion @jtpio 

> I don't have right on this repository. So I let you merge it.

Should we archive the project?

**Comment by jtpio** [comment]
Sounds good :+1: 

Merging and archiving the repo right after.

**Pull request merged**